src/form/bf.jl

```diff
@@ -1,8 +1,9 @@
+
+
+
 ""
-function variable_mc_voltage(pm::LPLinUBFModel; kwargs...)
-    for cnd in _PMs.conductor_ids(pm)
-        _PMs.variable_voltage_magnitude_sqr(pm; cnd=cnd, kwargs...)
-    end
+function variable_mc_voltage(pm::LPLinUBFModel; n_cond::Int=3, nw::Int=pm.cnw, bounded = true)
```

Comment:
what is the purpose of n_cond? It is not used and not being passed on either.

Reply:
ok, removed